pkgs/development/tools/pypi2nix/default.nix

```diff
@@ -27,7 +27,7 @@ in stdenv.mkDerivation rec {
     click
     requests
   ];
-  buildInputs = [ python zip makeWrapper nix.out nix-prefetch-scripts ];
+  buildInputs = [ python zip makeWrapper nix.out nix-prefetch-scripts python3Packages.setuptools ];
```

Comment:
use `python.pkgs.setuptools` to guarantee the correct version of `setuptools` is used.